### PR TITLE
Add pagination to GetAddressRuneUTXOIndex endpoint

### PR DESCRIPTION
**Pull request by dcorral**

### CODE REVIEW

File: src/Methods/Runes/GetAddressRuneUTXOIndex.ts [hunk 1]
```diff
@@ -1,19 +1,25 @@
...
     runeTicker: string,
+    pagination: schema.pagination.optional(),
   }),
-  handler: async ({ address, runeTicker }): Promise<RuneUtxoBalance[]> => {
-    const balances = await runes.addressRunesUTXOs(address, runeTicker);
-    if (balances === null) {
-      throw new NotFoundError("Address not found");
-    }
-    return balances;
+  handler: async ({ address, runeTicker, pagination = {} }) => {
```
Comment (kevzzsk) [inline src/Methods/Runes/GetAddressRuneUTXOIndex.ts:16]: should we have a default pagination `limit` ?

File: src/Methods/Runes/GetAddressRuneUTXOIndex.ts [hunk 2]
```diff
@@ -1,19 +1,25 @@
...
+    const params: FindPaginatedParams<RuneUtxoBalance> = { ...pagination, filter: { address, runeTicker } };
```
Comment (danadi7) [inline src/Methods/Runes/GetAddressRuneUTXOIndex.ts:17]: Should we add a default sorting for pagination? I think without a sort key, the ordering of the results returned by MongoDB is not stable

**Review by kevzzsk — APPROVED** [review]

File: src/Methods/Runes/GetAddressRuneUTXOIndex.ts [hunk 3]
```diff
@@ -1,19 +1,30 @@
...
-    if (balances === null) {
-      throw new NotFoundError("Address not found");
-    }
-    return balances;
+  handler: async ({ address, runeTicker, pagination = {} }) => {
+    pagination.limit ??= 50;
+    const params: FindPaginatedParams<RuneUtxoBalance> = {
+      ...pagination,
+      filter: { address, runeTicker },
+      sort: { _id: "asc" },
```
Comment (danadi7) [inline src/Methods/Runes/GetAddressRuneUTXOIndex.ts:21]: can we set this default sorting only when the user does not supply a sort key?

**Review by danadi7 — APPROVED** [review]
LGTM!

**Pull request merged**